index.html

```diff
@@ -145,24 +145,24 @@ <h1 class="title">About</h1>
 
     <section class="contact">
       <div class="headline">
-      <h2 class="title">Get Started</h2>
-      <p class="myname-intro"> I'm always interested in hearing about new projects, 
-                               so if you'd like to chat please get
-                               in touch.</p>
-      <form action="https://formspree.io/f/xgebneoy" method="post" class="contact-form">
+        <h2 class="title">Get Started</h2>
+        <p class="myname-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      </div>
+
+      <form action="https://formspree.io/f/xgebneoy" method="post">
         <div class="form-row">
-          <label for="name">Your name</label>
-          <input type="text" id="name" name="name" maxlength="30" placeholder="Write your name..." required>
+           <label for="name">Your name</label>
+           <input type="text" id="name" name="name" maxlength="30" placeholder="Write Your name..." required>
         </div>
         <div class="form-row">
           <label for="email">Your email</label>
-          <input type="email" id="email" name="email" placeholder="Write your email..." required>
-        </div>
-        <div class="form-row">
-          <label for="contact-descrip">Your message</label>
-          <textarea name="contact-descrip" id="contact-descrip" cols="30" rows="10" maxlength="500" placeholder="Enter your message here..." required></textarea>
-        </div>
-        <button type="button" class="button-box">Start collaboration</button>
+          <input type="email" id="email" name="email" placeholder="Write Your email..." required>
+       </div>
+       <div class="form-row">
+        <label for="contact-descrip">Your message</label>
+        <textarea id="contact-descrip" name="contact-descrip" maxlength="500" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+       </div>
+         <button class="button-box" type="submit">Start collaboration</button>
       </form>
     </section>
```

Comment:
- [ ] You have done great so far. I would like to reiterate what the first reviewer pointed out. The design of the form needs to be identical to that of the Figma template that you chose. To achieve this, you need to add `padding` to the form inputs for _**text**_ and **_email_**, give little space between the **_form inputs and their labels_** and give a `margin-left` and `margin-right` of `18px` to the form. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/229894331-14903e5a-0bfa-484d-a637-25a72af3a8ec.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/229894454-ea38ddb1-92ab-4823-bc14-85858cfc194e.png)